README.md

```diff
@@ -1 +1 @@
-# hello-microverse
\ No newline at end of file
+
```

Comment:
Please make your readme descriptive and professional. You can use this [Readme template](https://github.com/microverseinc/readme-template) and edit it to be personalized, to your current project. If you find this confusing, you can reach me on slack with 'Kanu Mike Chibundu'.